CRM/Event/Form/Registration/Register.php

```diff
@@ -1196,8 +1197,9 @@ public static function checkRegistration($fields, $form, $isAdditional = FALSE)
             if ($form->_pcpId) {
               $registerUrl .= '&pcpId=' . $form->_pcpId;
             }
-
-            $status = ts("It looks like you are already registered for this event. If you want to change your registration, or you feel that you've received this message in error, please contact the site administrator.") . ' ' . ts('You can also <a href="%1">register another participant</a>.', [1 => $registerUrl]);
+            $registrationType = (CRM_Event_PseudoConstant::getKey('CRM_Event_BAO_Participant', 'participant_status_id', 'On waitlist') == $participant->status_id) ? 'waitlisted' : 'registered';
```

Comment:
Seeing this late, but this is a good example of an incorrect use of "ts". It's impossible to translate a string like this, even if the individual words were in "ts" (because of grammar).

Reply:
https://github.com/civicrm/civicrm-core/pull/19027